### PR TITLE
OZW usercode documentation

### PR DESCRIPTION
**Pull request by firstof9**
## Proposed change

Add documentation for ozw.set_usercode, ozw.get_usercode, and ozw.clear_usercode

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/37390
- Link to parent pull request in the Brands repository: n/a
- This PR fixes or closes issue: n/a

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by balloob — APPROVED** [review]

File: source/_integrations/ozw.markdown [hunk 1]
```diff
@@ -65,3 +65,24 @@ this operation.
...
 | `instance_id`          | no       | The OZW Instance/Controller to use, defaults to 1. |
+
+### Service `ozw.set_usercode`
+
+This service will set the usercode to X at code_slot Y.
+Valid usercodes are at least 4 digits.
+
+| Service Data Attribute | Required | Description                             |
+| ---------------------- | -------- | --------------------------------------- |
+| `entity_id`            | yes      | Lock entity to set the usercode.        |
```
Comment (frenck) [inline source/_integrations/ozw.markdown:76]: This service uses `async_register_entity_service` to register, thus the `entity_id` is optional as an area can be targeted instead.

File: source/_integrations/ozw.markdown [hunk 2]
```diff
@@ -65,3 +65,24 @@ this operation.
...
+| `usercode`             | yes      | The code to set in the slot.            |
+
+### Service `ozw.clear_usercode`
+
+This service will clear the usercode in code_slot X.
+Valid code_slots are 1-254.
+
+| Service Data Attribute | Required | Description                               |
+| ---------------------- | -------- | ----------------------------------------- |
+| `entity_id`            | yes      | Lock entity to clear the usercode.        |
```
Comment (frenck) [inline source/_integrations/ozw.markdown:87]: This service uses `async_register_entity_service` to register, thus the `entity_id` is optional as an area can be targeted instead.

File: source/_integrations/ozw.markdown [hunk 3]
```diff
@@ -65,3 +65,24 @@ this operation.
 | Service Data Attribute | Required | Description                                        |
 | ---------------------- | -------- | -------------------------------------------------- |
 | `instance_id`          | no       | The OZW Instance/Controller to use, defaults to 1. |
+
+### Service `ozw.set_usercode`
+
+This service will set the usercode to X at code_slot Y.
```
Comment (frenck) [inline source/_integrations/ozw.markdown:71]: "code_slot" can be written as normal words, or, if really needed to be a reference to the variable, it needs to be code fenced e.g.,: ``` `code_slot` ```

File: source/_integrations/ozw.markdown [hunk 4]
```diff
@@ -65,3 +65,24 @@ this operation.
...
+
+| Service Data Attribute | Required | Description                             |
+| ---------------------- | -------- | --------------------------------------- |
+| `entity_id`            | yes      | Lock entity to set the usercode.        |
+| `code_slot`            | yes      | The code slot to set the usercode into. |
+| `usercode`             | yes      | The code to set in the slot.            |
+
+### Service `ozw.clear_usercode`
+
+This service will clear the usercode in code_slot X.
```
Comment (frenck) [inline source/_integrations/ozw.markdown:82]: "code_slot" can be written as normal words, or, if really needed to be a reference to the variable, it needs to be code fenced e.g.,: ``` `code_slot` ```

File: source/_integrations/ozw.markdown [hunk 5]
```diff
@@ -65,3 +65,24 @@ this operation.
...
+Valid code_slots are 1-254.
```
Comment (frenck) [inline source/_integrations/ozw.markdown:83]: Missing word:

```
Valid `code_slots` are between 1-254.
```

Additionally, this comment can be part of the description of the service data attribute in the table above.

Currently, it is missing in the `ozw.set_usercode` description? Or doesn't it have that limitation?

File: source/_integrations/ozw.markdown (hunk 1 above)
Comment (firstof9) [reply source/_integrations/ozw.markdown:76]: I'll submit a PR with the requested changes.

File: source/_integrations/ozw.markdown (hunk 5 above)
Comment (firstof9) [reply source/_integrations/ozw.markdown:83]: I'll get a PR to fix the services.yaml too, thanks for that.

Comment (firstof9) [reply source/_integrations/ozw.markdown:83]: Here it is: https://github.com/home-assistant/home-assistant.io/pull/13960
Thank you for your review @frenck

**Pull request merged**